source/client/scripts/game.js

```diff
@@ -151,7 +167,14 @@ function generateCardsWithListeners(numCards) {
     cardEl.append(cardBackFaceEl, cardFrontFaceEl);
     cardContainerEl.append(cardEl);
 
-    cardContainerEl.addEventListener('click', cardClickHandler);
+    oracle.addEventListener('animationend', function activate() {
+      cardContainerEl.classList.add('active-container');
+      cardContainerEl.addEventListener('click', cardClickHandler);
+      oracle.removeEventListener('animationend', activate);
```

Comment:
You can add `{ once: true }` as an extra parameter to an event listener to avoid this remove syntax on the inside